### DISCUSSION TITLE
Geoblocking

### DISCUSSION THREAD

**Discussion by WEBDIENSTE** [TechnitiumSoftware/DnsServer#1082] [Q&A]
Hi!

I'd like to block dns requests from specific countries, but I couldn't find a proper description to archive that.
How do I block multiple countries from resolving dns requests?

There is an App, but is the server able to sync Apps to other dns technitium dns servers or must it configured on all servers individually?

**Comment by ShreyasZare** [comment]
Thanks for asking. There is no option to do that currently but it can be implemented. But, you will need to purchase MaxMind geolocation database for that to work. Are you willing to purchase that? Also note that such databases are not accurate so it may allow some users in that blocked region or block users from other regions.
